include/crpropa/module/Observer.h

```diff
@@ -281,6 +281,17 @@ class ObserverTimeEvolution: public ObserverFeature {
 	DetectionState checkDetection(Candidate *candidate) const;
 	std::string getDescription() const;
 };
+
```

Comment:
What's the difference compared to the `MaximumTrajectoryLength` module?

Reply:
The first difference is that is Observer based. In this case it is easier to combine different break conditions, having them in the same observer. In principle all functionality could be received with the `MaximumTrajectoryLength`. 

In the practical solution there are two small differences: 
1. The Observer implementation does not have the accept/reject flags, which might reduce the output, if all properties are enabled. 
2. The Observer implementation does not have the overhead with the `observerPositions`. In case without them, the size of the list is checked in every propagation step.

Reply:
I removed this feature as it does not bring a new functionality. Same can be done with `ObserverTimeEvolution` with a single time step.